### PR TITLE
Contact section added

### LINKED ISSUE #18: Add a Contact Us section
The contact section should include links to:
- LinkedIn
- Discord
- GitHub

### PR DESCRIPTION
**Pull request by jradha11**
I have added a section for contact details which includes our slack, discord, and GitHub links. This PR closes #18

### CODE REVIEW

**Comment by jradha11** [comment]
image
The section looks like this

**Comment by sjha2048** [comment]
let's make this draft PR as if we merge #17  this won't be relevant anymore

**Comment by jradha11** [comment]
A similar kind of layout could be used in the footer.

**Comment by jradha11** [comment]
@NidhiKJha review required

File: src/components/contact.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+          </div>
+
+          <div className="pbody">
+            <p>Want to join our community? You can find us on :</p>
+              <p>
+              <a href="https://discord.gg/B3Ce9YE" target="__blank" className="handle"><img src={discord} alt="discord" className="contact-icon"/> Discord</a> 
+              </p>
+              <p>
+              <a href="https://lnkd.in/gYac5Ci" target="__blank" className="handle"><img src={slack} alt="slack" className="contact-icon"/> Slack</a>
+              </p>
```
Comment (NidhiKJha) [inline src/components/contact.js:30]: Don't add discord and linkedin in two different paragraphs instead put them in a single line.

File: src/components/contact.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+              <p>
+              <a href="https://github.com/The-Codesis" target="__blank" className="handle"><img src={gh} alt="slack" className="contact-icon"/> GitHub</a>
+              </p>
```
Comment (NidhiKJha) [inline src/components/contact.js:33]: Remove github from here, we will add it in project section.

**Comment by jradha11** [comment]
@NidhiKJha Sure, I will make the changes to the PR. Thank you !

**Comment by jradha11** [comment]
@NidhiKJha I have made the suggested changes :)

**Comment by NidhiKJha** [comment]
Awesome! Thanks :)

**Pull request merged**